tests/fields/fields.go

```diff
@@ -17,13 +17,13 @@
 package main
 
 import (
+	"code.google.com/p/goauth2/oauth"
```

Comment:
@gmlewis okay, thanks. I force-pushed the change to keep it to just the single commit.